stream-compositions/services/legal-entity-composition-service/pom.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.backbase.buildingblocks</groupId>
+        <artifactId>service-sdk-starter-reactive</artifactId>
+        <version>13.4.0</version>
+        <relativePath/>
+    </parent>
+
+    <artifactId>legal-entity-composition-service</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+
+    <packaging>jar</packaging>
+
+    <name>Stream :: Compositions :: Services :: Legal Entity</name>
+
+    <properties>
+        <boat-maven-plugin.version>0.15.0</boat-maven-plugin.version>
+    </properties>
+
+    <dependencies>
+        <!-- Service SDK dependencies -->
+        <dependency>
+            <groupId>com.backbase.buildingblocks</groupId>
+            <artifactId>events</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.backbase.buildingblocks</groupId>
+            <artifactId>auth-security</artifactId>
```

Comment:
bear in mind that according to the [documentation](https://community.backbase.com/documentation/ServiceSDK/latest/service_sdk_reactive#_service_sdk_modules_that_dont_support_reactive_applications) this module doesn't support reactive applications, you can still secure this service leveraging spring though.

Reply:
I'm a bit sceptical about not using built-it backbase sdk mechanisms. It may be a bit confusing for developers how to use/integrate with this service